src/Modules/Microsoft.Azure.Functions.PowerShellWorker/Microsoft.Azure.Functions.PowerShellWorker.psm1

```diff
@@ -235,6 +235,8 @@ function New-DurableOrchestrationCheckStatusResponse {
     The TaskHubName of the orchestration instance that will handle the external event.
 .PARAMETER ConnectionName
     The name of the connection string associated with TaskHubName
+.PARAMETER AppCode
```

Comment:
These changes do not seem to be related to the declared PR goal. I will not be insisting on necessarily separating them, but I want us to be mindful of this.

Reply:
Good call. At the very least, I should have called this out. I'll keep an eye out in the future